app/commands/decidim/action_delegator/admin/create_delegation.rb

```diff
@@ -31,6 +32,16 @@ def call
 
         attr_reader :form, :performed_by
 
+        def above_max_grants?
+          return false unless grants_count >= form.setting.max_grants
+
+          true
+        end
```

Comment:
```suggestion
        def above_max_grants?
          grants_count >= form.setting.max_grants
        end
```
I think this can be simplified!